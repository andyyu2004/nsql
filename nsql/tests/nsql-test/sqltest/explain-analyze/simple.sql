CREATE TABLE integers (
    id int PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
    i int
);

INSERT INTO integers (i)
    VALUES (0),
    (1),
    (2),
    (NULL);

EXPLAIN ANALYZE TIMING OFF
SELECT
    id,
    i + 1
FROM
    integers
WHERE
    i <> 2;
