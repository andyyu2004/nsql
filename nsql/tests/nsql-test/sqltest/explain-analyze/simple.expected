 projection (integers.id, #15(integers.i, 1)) in=2 out=2 
   filter #6(integers.i, 2) in=4 out=2                   
     scan integers (id, i) in=0 out=4                    
                                                         